Assets/HoloToolkit/Input/Scripts/SelectedObjectMessageSender.cs

```diff
@@ -0,0 +1,46 @@
+﻿using HoloToolkit.Unity;
+using UnityEngine;
+
+/// <summary>
+/// SelectedObjectMessageSender class sends a Unity message to currently selected object.
+/// Object selection is controlled via OnSelectObject and OnClearSelection events.
+/// Object selection and messages can be triggered using keyword responses registered in KeywordManager.
+/// </summary>
+public class SelectedObjectMessageSender : MonoBehaviour
```

Comment:
@NeerajW I added more extensive comments about voice commands since the note about KeywordManager responses may not be clear.